File: benchmark/comparison/results/blind_ts/03.ts
function is_palindrome(s: string): boolean {
  const rev = s.split("").reverse().join("");
  return s === rev;
}

console.log(String(is_palindrome("racecar")));
console.log(String(is_palindrome("hello")));
console.log(String(is_palindrome("")));
console.log(String(is_palindrome("a")));

File: benchmark/comparison/results/blind_py/03.py
def is_palindrome(s):
    return s == s[::-1]

def fmt(b):
    return "true" if b else "false"

print(fmt(is_palindrome("racecar")))
print(fmt(is_palindrome("hello")))
print(fmt(is_palindrome("")))
print(fmt(is_palindrome("a")))
